pandas/io/formats/format.py

```diff
@@ -1095,7 +1106,7 @@ def format_array(
     space: Optional[Union[str, int]] = None,
     justify: str = "right",
     decimal: str = ".",
-    leading_space: Optional[bool] = None,
+    leading_space: bool = "compat",
```

Comment:
this isn't a boolean? Can you run mypy on your changes

Reply:
oh, yeah, didn't see your comment when I committed my latest change. I changed already